licenses.d2iq.yaml

```diff
@@ -77,6 +77,12 @@ resources:
         ref: v${image_tag}
         license_path: LICENSE
         notice_path: NOTICE.md
+  - container_image: docker.io/grafana/grafana:9.4.3
```

Comment:
Why do we have grafana updates for Kubecost bump PR 🤔 , Is it due to rebase?

Reply:
kubecost deploys its own instance of grafana